app/controllers/hooks_controller.rb

```diff
@@ -9,9 +9,15 @@ def pulls
     SyncPullsJob.perform_later(user, repo)
 
     # 現状は問題ないが、リクエストが増えた際にバックグラウンドに回す方が良いかも
-    if request.headers["HTTP_X_GITHUB_EVENT"] == "pull_request" && params[:pull_request][:merged]
-      pull = Pull.find_by(id: params[:pull_request])
-      pull&.review_requests.update_all(state: :resolved, is_open: false)
+    if request.headers["HTTP_X_GITHUB_EVENT"] == "pull_request" && params[:action] == "closed"
+      pull = Pull.find_by(id: params[:pull_request][:id])
+      return render json: { message: "no pulls" } unless pull
+
+      if params[:pull_request][:merged]
+        pull.review_requests.update_all(state: :resolved, is_open: false)
+      else
+        pull.review_requests.update_all(is_open: false)
```

Comment:
__[RuboCop]__

Rails/SkipsModelValidations: Avoid using `update_all` because it skips validations.


<!-- Comment by [Sider](https://sider.review) -->